pkgs/tools/misc/szyszka/default.nix

```diff
@@ -42,6 +42,18 @@ rustPlatform.buildRustPackage rec {
     Foundation
   ]);
 
+  preBuild = ''
+    mkdir -p $out/share/{applications,icons/hicolor/scalable/apps,metainfo}
+
+    install -m 444 \
+        -D $src/data/com.github.qarmin.szyszka.desktop \
```

Comment:
We usually don't reference $src because current directory is a mutable copy of src. Unless they got removed in build process...